### PR TITLE
Fix/to value decimal display

### PR DESCRIPTION
**Pull request by Kevingislason**
## Description

closes: #390 

- Prevents users from inputing more decimals than are possible for a given asset on the Bridge and ZkTransact pages
- Displays the amount to be sent without rounding on the ZkTransact page

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [x] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [x] Updated relevant documentation in the code
- [x] Test code changes manually, and describe your procedure in a comment on this PR
- [x] Mark this PR with the correct milestone

### CODE REVIEW

File: src/utils/validation/handleChangeBalanceInput.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+  const decimalIndex = balanceString.indexOf('.');
+  if (decimalIndex === -1) {
+    return false;
+  }
+  const decimalPlaces = balanceString.length - decimalIndex - 1;
+  return decimalPlaces > assetType.numberOfDecimals;
+};
+
+const handleChangeBalanceInput = (
```
Comment (zhangjing28) [inline src/utils/validation/handleChangeBalanceInput.ts:18]: Nit: Maybe it's better define a function with many arguments like this:
```javascript
const handleChangeBalanceInput = (
{newInputString, preInputString, setInputString, setBalance, assetType}: {
  newInputString: string,
  prevInputStrig: string,
  setInputString: (inputString: string) => void,
  setBalance:  (balance: Balance | null) => void,
  assetType: AssetType})
```
I think it's easier for us to know the meaning of each argument, easier to add more arguments, easier to call the function without considering the argument's order

**Review by zhangjing28 — APPROVED** [review]
LGTM

**Comment by jincan39** [comment]
LGTM

File: src/utils/validation/handleChangeBalanceInput.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  newInputString: string,
+  prevInputStrig: string,
+  setInputString: (inputString: string) => void,
+  setBalance:  (balance: Balance | null) => void,
```
Comment (jincan39) [inline src/utils/validation/handleChangeBalanceInput.ts:22]: image
image

eslint-plugin shows error, think we can turn 'no-unused-vars' to 'warn' or skip check by pattern

Comment (Kevingislason) [reply src/utils/validation/handleChangeBalanceInput.ts:22]: Good idea, thanks

**Comment by jincan39** [comment]
LGTM

**Review by jincan39 — APPROVED** [review]

**Review by zhangjing28 — APPROVED** [review]

**Pull request merged**